### PR TITLE
interactive google CLI instructions

### PR DESCRIPTION
**Pull request by choldgraf**
This adds interactive google shell instructions now that their interface got updated and actually works! It splits a section in the kubernetes setup for google cloud into two sub-points so that users can choose either interactive CLI or their own shell.

### CODE REVIEW

**Comment by yuvipanda** [comment]
This LGTM, @choldgraf :)

If you rewrite the PR description to work standalone I am happy to merge this :)

**Comment by yuvipanda** [comment]
(and in general, PR descriptions should IMO always work standalone and be written lime commit descriptions. Add a comment to spur discussions)

**Comment by choldgraf** [comment]
updated the top-level comment

File: doc/source/create-k8s-cluster.rst [hunk 1]
```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
...
+   for each are detailed below:
 
-   - Go to the `gcloud downloads page <https://cloud.google.com/sdk/downloads>`_
-     to **download and install the gcloud SDK**.
-   - See the `gcloud documentation <https://cloud.google.com/sdk/>`_ for
-     more information on the gcloud SDK.
-   - Install ``kubectl``, which is a tool for controlling kubernetes. From
-     the terminal, enter:
+   * **Google Interactive CLI**. Start the Google Cloud interactive command line
+     by Clicking the button shown below. This will start an interactive shell
```
Comment (willingc) [inline doc/source/create-k8s-cluster.rst]: clicking

File: doc/source/create-k8s-cluster.rst [hunk 2]
```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
...
+     session within Google Cloud.
+
+   .. image:: _static/images/google/start_interactive_cli.png
+      :align: center
+
+   * **Working locally**. Install and initialize the **gcloud command-line tools**.
```
Comment (willingc) [inline doc/source/create-k8s-cluster.rst]: **Install and use the gcloud command-line tool**

Link to: https://cloud.google.com/pubsub/docs/quickstart-cli

File: doc/source/create-k8s-cluster.rst [hunk 3]
```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
...
+   interactive CLI, or working locally from your own shell interface. Instructions
+   for each are detailed below:
 
-   - Go to the `gcloud downloads page <https://cloud.google.com/sdk/downloads>`_
-     to **download and install the gcloud SDK**.
-   - See the `gcloud documentation <https://cloud.google.com/sdk/>`_ for
-     more information on the gcloud SDK.
-   - Install ``kubectl``, which is a tool for controlling kubernetes. From
-     the terminal, enter:
+   * **Google Interactive CLI**. Start the Google Cloud interactive command line
```
Comment (willingc) [inline doc/source/create-k8s-cluster.rst]: **Use the Google Cloud Shell**. Start Google Cloud Shell by clicking ...

https://cloud.google.com/shell/docs/

File: doc/source/create-k8s-cluster.rst [hunk 4]
```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
 
 2. Enable the `Kubernetes Engine API <https://console.cloud.google.com/apis/api/container.googleapis.com/overview>`_.
 
-3. Install and initialize the **gcloud command-line tools**. These tools send
-   commands to Google Cloud and lets you do things like create and delete
-   clusters.
+3. Prepare your command line interface. You have two options, using Google's
```
Comment (willingc) [inline doc/source/create-k8s-cluster.rst]: 3. Use your preferred command line interface.

You have two options: a) use the Google Cloud Shell (no installation needed) or b) install and use the gcloud command-line tool.  If you are unsure which to choose, we recommend beginning with option "a" and use Google Cloud Shell.

File: doc/source/create-k8s-cluster.rst (hunk 2 above)
Comment (willingc) [reply doc/source/create-k8s-cluster.rst]: Consistently refer to gcloud command-line tool.

File: doc/source/create-k8s-cluster.rst [hunk 5]
```diff
@@ -38,22 +38,33 @@ connect your credit card or other payment method to your google cloud account.
...
+       more information on the gcloud SDK.
+4. Install ``kubectl``, which is a tool for controlling kubernetes. From
+   the terminal, enter:
 
      .. code-block:: bash
 
         gcloud components install kubectl
 
-4. Create a Kubernetes cluster on Google Cloud, by typing in the following
+5. Create a Kubernetes cluster on Google Cloud, by typing in the following
```
Comment (willingc) [inline doc/source/create-k8s-cluster.rst]: ...typing the following command into either the Google Cloud shell or the  gcloud command-line tool:

**Comment by choldgraf** [comment]
comments addressed @willingc !

**Pull request merged**